### PR TITLE
Update price-points logic.

### PR DESCRIPTION
**Pull request by nanek**
@CarsonBanov 

/cc @JuanCaicedo

### CODE REVIEW

**Comment by nanek** [comment]
Is there existing logic to break this into orders?

File: test/formatter.js [hunk 1]
```diff
@@ -61,4 +61,14 @@ describe('formatter', function(){
     });
 
   });
+
+  describe('generatePricePoints', function(){
+    it('should generate price points', function(){
+      var prices = formatter.generatePricePoints();
+
+      expect(prices[0]).to.eql('0.00');
```
Comment (JuanCaicedo) [inline test/formatter.js]: Should this include 0.00? I would think it should start at 0.01

**Comment by JuanCaicedo** [comment]
@nanek I don't think there is, the price points were in 5 cent increments before and we would create all of them for each order (as is for one order, +1 cent for the next, +2 for the next, etc)

**Comment by JuanCaicedo** [comment]
I doubt you'll be able to create all 10000 line items in one node script, 400 is already a struggle. You might need some type of logic to split them up in your `scripts/` script .

File: lib/formatter.js [hunk 2]
```diff
@@ -319,10 +319,25 @@ function formatOrder(name, traffickerId, partner) {
...
+/**
+ * Generate price points.
+ *
+ * @return {Array}              Price points.
+ */
+var generatePricePoints = function() {
+  var points = [];
+  points.push("0.00");
+  for (var i=1; i<=10000; i++) {
+    points.push((i/100).toFixed(2).toString());
```
Comment (CarsonBanov) [inline lib/formatter.js]: I don't think you need `.toString()` https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed

File: lib/formatter.js [hunk 3]
```diff
@@ -319,10 +319,25 @@ function formatOrder(name, traffickerId, partner) {
...
 }
 
+/**
+ * Generate price points.
+ *
+ * @return {Array}              Price points.
+ */
+var generatePricePoints = function() {
+  var points = [];
+  points.push("0.00");
```
Comment (CarsonBanov) [inline lib/formatter.js]: Maybe should be removed, see Juan's comment in the tests about starting with 0.01

**Comment by nanek** [comment]
Updated.

File: scripts/create-line-items.js [hunk 4]
```diff
@@ -34,22 +33,18 @@ var credentials = {
...
-var channel = argv.channel;
-var region = argv.region;
 var position = argv.position;
 var partner = argv.partner;
 var platform = argv.platform;
 var offset = argv.offset;
 
 // use arguments to determine any other variables
-var pricePoints = require('./price-points');
+var pricePoints = formatter.generatePricePoints(1, 400);
```
Comment (CarsonBanov) [inline scripts/create-line-items.js:42]: Could this `1` and `400` be another argument to the script?

**Comment by CarsonBanov** [comment]
One minor question (not sure I'm really understanding usage properly), but Approved

File: scripts/create-line-items.js (hunk 4 above)
Comment (nanek) [inline scripts/create-line-items.js:42]: Yes. Adding this later.

**Comment by nanek** [comment]
Closing to merge against update-prebid.

**Pull request closed without merging**